recipe/yum_requirements.txt

```diff
@@ -0,0 +1,16 @@
+liburiparser-devel
```

Comment:
> No, you need to rerender everytime you change the file.

Yep. I also recommend you to get the docker image and test the build locally so you can iterate quicker.

```shell
docker pull condaforge/linux-anvil   # Downloads the image.
docker run -it --rm -v $(pwd):/home/ condaforge/linux-anvil /bin/sh  # Starts an interactive session.
```

That way you can `yum install`/`uninstall` things and build until you get a combination that works.